src/Symfony/Component/Serializer/Tests/Annotation/MaxDepthTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Tests\Annotation;
+
+use Symfony\Component\Serializer\Annotation\MaxDepth;
+
+/**
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+class MaxDepthTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @expectedException \Symfony\Component\Serializer\Exception\InvalidArgumentException
+     */
+    public function testNotAnIntMaxDepthParameter()
+    {
+        new MaxDepth(array('value' => 'coopTilleuls'));
```

Comment:
I know I've told you this before, but using the name of a real company as the value of the tests looks wrong to me and is inconsistent with the rest of Symfony codebase.